Makefile

```diff
@@ -12,7 +12,8 @@ test:
 .PHONY: test
 
 integration-test:
-	./hack/integration-test.sh
```

Comment:
The file name for the script was wrong anyway. 😄

Reply:
Yeah actually I need https://github.com/openshift/cluster-version-operator/pull/1203 first so that I can merge https://github.com/openshift/release/pull/65742 and then the tests on _this_  PR should be green